src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/applicationinfo.cpp

```diff
@@ -213,7 +210,7 @@ APPLICATION_INFO::TryCreateHandler(
 }
 
 VOID
-APPLICATION_INFO::ShutDownApplication(bool fServerInitiated)
+APPLICATION_INFO::ShutDownApplication(const bool fServerInitiated)
```

Comment:
?

Reply:
`fServerInitiated` isn't getting changed and not intended to get muted and so making it `const`

Reply:
I think it because it's a bool, making a bool const doesn't help much as the size of what's passed in. Right @BrennanConroy

Reply:
I don't think this is a big deal though to block merging.

Reply:
Yeah, although it does make it clear that no one should modify it in the function (not super important)